src/routes/Viewer.vue

```diff
@@ -48,6 +63,11 @@ export default defineComponent({
     }
     this.globalStore.progress = false;
   },
+  methods: {
+    launchSimul() {
```

Comment:
Don't use abbreviations
```suggestion
    launchSimulator() {
```

Reply:
Sry